airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -281,6 +283,7 @@ public void testDiscoverSourceSchema() throws ApiException {
         .build());
     final AirbyteStream stream = new AirbyteStream()
         .name(STREAM_NAME)
```

Comment:
should we unset this?

Reply:
No, this is required as we are currently backward compatible and everything keeps working as it was before.

The new AirbyteStreamName field is not actually influencing the stream names for the moment